tools/test-help-documentation

```diff
@@ -25,14 +29,21 @@ os.makedirs('var/help-documentation', exist_ok=True)
 LOG_FILE = 'var/help-documentation/server.log'
 external_host = "localhost:9981"
 
-with test_server_running(options.force, external_host, log_file=LOG_FILE,
-                         dots=True, use_db=True):
-    ret_help_doc = subprocess.call(('scrapy', 'crawl_with_status', 'help_documentation_crawler'),
-                                   cwd='tools/documentation_crawler')
-    ret_api_doc = subprocess.call(('scrapy', 'crawl_with_status', 'api_documentation_crawler'),
-                                  cwd='tools/documentation_crawler')
-    ret_portico_doc = subprocess.call(('scrapy', 'crawl_with_status', 'portico_documentation_crawler'),
+if not options.skip_external_link_check:
```

Comment:
I think it'd be better to make a varaible, `extra_args`, that we pass into the 3 calls., rather than duplicating so much code.